File: app/src/play/java/com/ringdroid/StoragePermissionUtils.java
package com.ringdroid;

import android.app.Activity;

public class StoragePermissionUtils {

    public static boolean hasExternalStoragePermission() {
        // Disabled for Play flavor
        return false;
    }

    public static void requestExternalStoragePermission(Activity activity) {
        // Intentionally no-op
    }
}

File: app/src/play/java/com/ringdroid/StoragePermissionUtils.kt
package com.ringdroid

import android.app.Activity

open class StoragePermissionUtils {
    fun hasExternalStoragePermission(): Boolean {
        // Disabled for Play flavor
        return false
    }

    fun requestExternalStoragePermission(activity: Activity?) {
        // Intentionally no-op
    }
}
